I'm sorry, but an uncaught exception occurred.

While running game code:
  File "renpy/common/00start.rpy", line 237, in script call
    call _splashscreen from _call_splashscreen_1
  File "game/script.rpy", line 329, in script
    python:
  File "game/script.rpy", line 447, in <module>
    audio_file_path = os.path.join(audio_folder,myuuid+".mp3")
NameError: name 'myuuid' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "renpy/common/00start.rpy", line 237, in script call
    call _splashscreen from _call_splashscreen_1
  File "game/script.rpy", line 329, in script
    python:
  File "/Library/Afolder/renpy-8.1.3-sdk/renpy/ast.py", line 1138, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "/Library/Afolder/renpy-8.1.3-sdk/renpy/python.py", line 1122, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "game/script.rpy", line 447, in <module>
    audio_file_path = os.path.join(audio_folder,myuuid+".mp3")
NameError: name 'myuuid' is not defined

macOS-14.1.1-arm64-arm-64bit arm64
Ren'Py 8.1.3.23091805
Amadeus 1.2.9.4
Sun Nov 12 14:13:00 2023
